### PR TITLE
Add support for `FileManagementService`

### PR DESCRIPTION
**Pull request by geel9**
- Adds bindings for `FileManagementService`
- Adds bindings for `AccessManagementService` where missing

### CODE REVIEW

**Review by fundthmcalculus — APPROVED** [review]
Nice!

File: go/services/file_management_service.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
+package services
+
+import "github.com/trinsic-id/sdk/go/proto/services/filemanagement/v1/filemanagement"
```
Comment (fundthmcalculus) [inline go/services/file_management_service.go:4]: You need to import `context.Context` up here. Look at any other service for an example

Comment (geel9) [reply go/services/file_management_service.go:4]: hmm, I had done that before running `protoc-gen-sdk` -- this is why auto-removing unused imports on save is annoying

Comment (fundthmcalculus) [reply go/services/file_management_service.go:4]: That's `gofmt` for you. It's opinionated for the sake of being opinionated. It's also annoying as heck.

**Pull request merged**